pkg/operation/botanist/controlplane/kubescheduler/kube_scheduler.go

```diff
@@ -126,13 +126,6 @@ func (k *kubeScheduler) Deploy(ctx context.Context) error {
 		service    = k.emptyService()
 		deployment = k.emptyDeployment()
 
-		labels = map[string]string{
```

Comment:
Can you also adapt it for the `clusterautoscaler` pkg?

Reply:
yes, that was done already.